Git/config.sublime-syntax

```diff
@@ -0,0 +1,163 @@
+%YAML 1.2
+---
+# Syntax based on documentation here:
+# https://git-scm.com/docs/git-config#_syntax
+
+name: Git Config
+file_extensions:
+  - gitconfig                 # /etc/gitconfig
+  - .gitconfig                # ~/.gitconfig
+first_line_match: ^\[core\]   # .git/config files always start with [core]
+scope: source.gitconfig
+
+variables:
+  section_name: '[\w.-]+(?=[\s\]])'
+  space_or_eol: (?=(?:\s|$))  # makes values on very last line match properly
+  variable_name: '[a-zA-Z][\w-]*'
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - include: section
+    - include: key-value-pair
+
+  boolean:
+    # 0 and 1 omitted due to mismatches (e.g. "log -1", "HEAD~1"). will be
+    # matched as unquoted strings instead
+    - match: \b(?:yes|true|on|off|no|false){{space_or_eol}}
+      scope: constant.language.gitconfig
+
+  color-attribute:
+    - match: \b(?:no-?)?(?:ul|strike|reverse|italic|dim|bold|blink){{space_or_eol}}
+      scope: support.constant.color-attribute.gitconfig  # TODO: correct scope?
+
+  color-hex:
+    - match: (#)\h{6}
+      captures:
+        0: constant.other.color.rgb-value.gitconfig
+        1: punctuation.definition.constant.gitconfig
+
+  color-named:
+    - match: \b(?:yellow|white|red|normal|magenta|green|cyan|blue|black|auto){{space_or_eol}}
+      scope: support.constant.color.gitconfig
+
+  comments:
+    - match: (#|;).*
+      captures:
+        1: punctuation.definition.comment.gitconfig
+      scope: comment.line.gitconfig
+
+  # '\b', '\n', '\t', '\"', '\\'
+  escape:
+    - match: \\[\\"bnt]
+      scope: constant.character.escape.gitconfig
+
+  illegal-newline:
+    - match: \n
+      scope: invalid.illegal.gitconfig
+      pop: true
+
+  # key = val
+  key-value-pair:
+    - match: ^\s*({{variable_name}})\s*(\=)
+      captures:
+        1: meta.key.gitconfig variable.language.gitconfig  # TODO: correct scope?
```

Comment:
I don't think this should be a variable.language, maybe variable.other.readwrite is more suitable (although most color schemes don't highlight that, but they'll have to catch up). You could also drop the meta.key scope.